### ISSUE TITLE
CI: Failed Travis jobs due to GBQ

### ISSUE THREAD

**Issue by dsaxton** [pandas-dev/pandas#36436]
Lots of failed jobs coming from Travis lately with tracebacks like this:

```python
    @pytest.fixture()
    def gbq_dataset(self):
        # Setup Dataset
        _skip_if_no_project_id()
        _skip_if_no_private_key_path()
    
        dataset_id = "pydata_pandas_bq_testing_" + generate_rand_str()
    
>       self.client = _get_client()

pandas/tests/io/test_gbq.py:160: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
pandas/tests/io/test_gbq.py:68: in _get_client
    return bigquery.Client(project=project_id, credentials=credentials)
../../../miniconda3/envs/pandas-dev/lib/python3.7/site-packages/google/cloud/bigquery/client.py:179: in __init__
    project=project, credentials=credentials, _http=_http
../../../miniconda3/envs/pandas-dev/lib/python3.7/site-packages/google/cloud/client.py:250: in __init__
    Client.__init__(self, credentials=credentials, client_options=client_options, _http=_http)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <google.cloud.bigquery.client.Client object at 0x7f5c46e157d0>
credentials = <google.oauth2.service_account.Credentials object at 0x7f5c46e15bd0>
_http = None
client_options = ClientOptions: {'api_endpoint': None, 'client_cert_source': None}

    def __init__(self, credentials=None, _http=None, client_options=None):
        if isinstance(client_options, dict):
            client_options = google.api_core.client_options.from_dict(client_options)
        if client_options is None:
            client_options = google.api_core.client_options.ClientOptions()
    
>       if credentials and client_options.credentials_file:
E       AttributeError: 'ClientOptions' object has no attribute 'credentials_file'
```

Line that's raising:

https://github.com/pandas-dev/pandas/blob/c8b44dda08ab3a2feb2c4572be960469c3f065f7/pandas/tests/io/test_gbq.py#L160

Sample Travis log:

https://api.travis-ci.org/v3/job/728073325/log.txt

cc @tswast if any ideas

**Comment by VirosaLi** [comment]
See https://github.com/googleapis/google-cloud-python/issues/10471.

**Comment by tswast** [comment]
@busunkim96 Perhaps there was another bad google-auth release?

**Comment by busunkim96** [comment]
`google-cloud-core==1.4.0` was released with an insufficient pin on `google-api-core`. This is resolved in `google-cloud-core>=1.4.1`. https://github.com/googleapis/python-cloud-core/blob/master/CHANGELOG.md#bug-fixes
